addons/l10n_in/report/account_invoice_report.py

```diff
@@ -232,8 +232,10 @@ def _sub_select(self):
                     ELSE (CASE WHEN aml.tax_base_amount <> 0 THEN aml.tax_base_amount ELSE NULL END)
                     END AS price_total,
                 (CASE WHEN aj.type = 'sale' AND (ai.type IS NULL OR ai.type != 'out_refund') THEN -1 ELSE 1 END) AS amount_sign,
-                at.id AS tax_id,
-                at.amount AS tax_rate
+                (CASE WHEN atr.parent_tax IS NOT NULL THEN atr.parent_tax
+                    ELSE at.id END) AS tax_id,
+                (CASE WHEN atr.parent_tax IS NOT NULL THEN parent_at.amount
+                    ELSE at.amount END) AS tax_rate
```

Comment:
@jva-odoo Is not the amount on the child tax?

Reply:
we need to group by the parent tax amount
like if parent tax is GST 5% and two children tax SGST 2.5% and CGST 2.5% then we need group by GST 5% so we do like this.

Reply:
The percentage/amount field on the parent tax is not supposed to be filled in. (in the case of type group)  It is weird to us.  There really is no other way?

Reply:
No other way 😞